### PR TITLE
Stack: Change step two of the cloud shell demo

### PR DESCRIPTION
**Pull request by dherault**
## Summary

image

4 possible command type:
- Single application 
- Complete stack
- Multiple applications or incomplete stack --> quick stack
- Error --> default to the full stack or Airbyte to at least display something

Resolves ENG-697
Resolves ENG-693

## Checklist

- [x] If required, I have updated the Plural documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] I have added a meaningful title and summary to convey the impact of this PR to a user.
- [x] I have added relevant labels to this PR to help with categorization for release notes.

### CODE REVIEW

**Comment by floreks** [comment]
@dherault all checks have failed. Probably just a linter issue.

**Comment by dherault** [comment]
Thanks Marcin,
resolved!

**Comment by michaeljguarino** [comment]
I'm seeing a crash when i try to visit the cloud shell from my personal gmail's user (at least on the firebase pr-deployment)

**Comment by dherault** [comment]
Yeay, you'd need a custom registred user, not google-based

**Comment by michaeljguarino** [comment]
It's also broken for your user you sent me

**Comment by maciaszczykm** [comment]
This seems to be related to the issue I reported yesterday on Slack. @floreks already provided [a fix for that](https://github.com/pluralsh/plural/pull/570) in the `stacks` branch. I'll merge it here.

**Comment by maciaszczykm** [comment]
@dherault I've merged `stacks` branch here to add recent fixes.

File: www/src/components/shell/CreateRepositoryCard.js [hunk 1]
```diff
@@ -23,52 +23,55 @@ const providerToDisplayName = {
...
+          {data?.scmAuthorization?.map(({ provider, url }) => (
+            <OnboardingCardButton
+              key={provider}
+              onClick={() => {
               // START <<Remove this after dev>>
-              if (process.env.NODE_ENV !== 'production' && DEBUG_SCM_TOKENS[provider]) {
-                console.log('going to ', `/oauth/callback/${provider.toLowerCase()}/shell?code=abcd`)
-                window.location = `/oauth/callback/${provider.toLowerCase()}/shell?code=abcd`
+                if (process.env.NODE_ENV !== 'production' && DEBUG_SCM_TOKENS[provider]) {
+                  console.log('going to ', `/oauth/callback/${provider.toLowerCase()}/shell?code=abcd`)
```
[the remark this answers is not in the document]
Comment (dherault) [reply www/src/components/shell/CreateRepositoryCard.js:38]: We really need it for onboarding development.

**Comment by maciaszczykm** [comment]
@dherault There is a new `package-lock.json` file.

**Review by maciaszczykm — APPROVED** [review]

**Review by floreks — APPROVED** [review]

**Pull request merged**